### DISCUSSION TITLE
Migration from Postman

### DISCUSSION THREAD

**Discussion by Suniron** [usebruno/bruno#375] [Q&A]
Hey there!

Is there a proper way to migrate from a Postman collection and environments to Bruno?

I really want to integrate Bruno in my team stack but I'm afraid about the migration of ours 300+ requests 😅.

When I tried ThunderClient (RIP since the paid version for sync) a few months ago and there was a Postman importation feature.

Otherwise, I'll try with ChatGPT 🤫.

Thanks for help.

**Comment by helloanoop** [comment]
We have support for importing postman collection. You ca try importing and let us know if you face any difficulties and which areas of migration needs improvement.

**Comment by pove** [comment]
@helloanoop, do you think it's possible to add an option also to export from Bruno to Postman? It is hard to convince to all the people and teams to use Bruno from the beginning. It is a slow process to migrate every person to Bruno -even more if in this "early" stage when version is under 1.0 and in Windows you have to download and install new releases practically every day.

To have an option to export to Postman could give the possibility to a part of the team to use Bruno from the beginning and force other to gradually migrate, without get into the prohibition of "you cannot use Bruno because not everyone is using it".

**Reply by helloanoop** [reply]
Yes, Can you create a GitHub issue. We can allow export to both formats Bruno Collection and Postman Collection.

**Comment by deepak-s-s** [comment]
hey @helloanoop  i have a large collection which i am looking to export from postman to bruno. But this is getting stuck and bruno closes itself. No able to do this. Im expecting this to be because of the size of the collection it was having 35Mb. Is there a way i can do this migration of large collections. ?
